package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "nashviva",
+  "version": "1.0.0",
+  "description": "A digital pocketknife for life in Nashville",
+  "scripts": {
+    "lint": "standard --verbose | snazzy",
+    "fixcode": "standard --fix",
+    "precommit": "if [ -d 'ignite-base' ]; then cd ignite-base; fi; npm run lint"
```

Comment:
Ahh yeah, I meant to set that up with [husky](https://www.npmjs.com/package/husky).